clients/py/v3io_frames/pdutils.py

```diff
@@ -28,23 +28,27 @@ def concat_dfs(dfs, frame_factory=pd.DataFrame, concat=pd.concat):
     # See https://stackoverflow.com/a/44086708/7650
     align_categories(dfs)
 
-    unique_names_set = set()
-    for df in dfs:
-        if hasattr(df.index, 'names'):
-            indices = list(df.index.names)
-            unique_names_set.update(indices)
+    had_index = True
+    if backend != 'tsdb':
```

Comment:
Better add a comment explaining what is the difference between the backends and why the seperate cases